fapi-client/src/main/scala/com/gu/facia/api/utils/FaciaContentImplicits.scala

```diff
@@ -0,0 +1,119 @@
+package com.gu.facia.api.utils
+
+import com.gu.contentapi.client.model.{Element, Tag, Content}
+import com.gu.facia.api.models.{ImageReplace, ImageCutout, FaciaContent}
+import org.joda.time.DateTime
+
+import scala.util.Try
+
+object FaciaContentImplicits {
+
+  implicit class FaciaContentImplicit(fc: FaciaContent) {
```

Comment:
@adamnfish Haha, good question. The record syntax makes this a bit less painful, and you have no `implicits` so the only way is the pure way; you don't really have a choice. At least in Haskell it would be slightly more concise in that you don't have to use some object with methods on it to access a method to pass something in, you just have the property sort of floating around for the type, kind of like implicits but in reverse syntax and without the dot.

I don't think this is multiple ways of doing the same thing at all; it's just a different way to use the underlying methods. This is how I'm using it everyone in `frontend`.

I don't personally don't like named imports, such as `import FaciaContentUtils.isLive`, to use `isLive(faciaContent)`, and I don't like having to use the fully qualified path for the method everywhere either `FaciaContentUtils.isLive(faciaContent)`.